oaknational/Oak-Web-Application
# Feat/lesq 747/skip filters [LESQ-747]

## Description

Music year: 2013

- Add button to skip filters to main content
- update search page and tests to use theme provider from oak components

## How to test

1. Go to https://deploy-preview-2386--oak-web-application.netlify.thenational.academy/search?term=waves
2.  Tab to skip filters button and click enter
3. You should be moved into the results, and tabbing will take you through the results

## Screenshots

How it should now look:
<img width="518" alt="Screenshot 2024-04-24 at 14 43 09" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/99b4dd7a-937e-4d9f-bf6b-161c98afc2c1">
